module-with-tests/.gitignore

```diff
@@ -0,0 +1,34 @@
+.eslintcache
+.cache
+scalastyle-output.xml
+.classpath
+.idea
+!.idea/vcs.xml
+.metadata
```

Comment:
let's move this to the root directory

Reply:
moved .metadata from modules **module-with-tests** and **module-with-no-tests** to **test-project-parent/.gitignore**

Reply:
Is that the only thing ? Can I merge the PR ?